service/src/main/java/org/kiwiproject/champagne/model/Tag.java

```diff
@@ -0,0 +1,26 @@
+package org.kiwiproject.champagne.model;
+
+import javax.validation.constraints.NotBlank;
+import java.time.Instant;
+
+import lombok.Builder;
+import lombok.Value;
+import lombok.With;
+
+/**
+ * Representation of a tag to link Hosts and Components.
+ */
+@Builder
+@Value
+public class Tag {
+
+    Long id;
```

Comment:
It would be more work, but you could use the kiwi validation groups for id, createdAt, and updatedAt such that they need to be null on insert and present on update. Consider for later perhaps.

Reply:
Created #287 to look at this later